src/sentry/snuba/sessions_v2.py

```diff
@@ -421,6 +425,14 @@ def _run_sessions_query(query):
     `totals` and again for the actual time-series data grouped by the requested
     interval.
     """
+
+    # We only return the top-N groups, based on the first field that is being
+    # queried, assuming that those are the most relevant to the user.
+    # In a future iteration we might expose an `orderBy` query parameter.
+    orderby = [f"-{query.primary_column}"]
```

Comment:
It seems a bit arbitrary to let the first specified field decide what groups will be returned, but I don't have a better idea.

Reply:
Hmm for some reason I was under the impression we were going to use the `totals` field for that?